recipe/meta.yaml

```diff
@@ -7,12 +7,12 @@ package:
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: b276739a5f863ccacb61999a3067d0895ee291c95502929b2ae56ea1f882e888
+  fn: {{ version }}.tar.gz
+  url: https://github.com/scikit-learn/scikit-learn/archive/{{ version }}.tar.gz
+  sha256: 3ca2993acf49c43fc6376912d43714cbee14e14c1d578f5bf422053eda19c206
 
 build:
-  number: 201
+  number: 1
```

Comment:
This should be 200.

ref: https://conda-forge.org/docs/blas.html

Reply:
I believe you. That doc could use a short paragraph of introduction though...

Reply:
That's fair. It was originally a spec Phil Elson and I drafted. Agree it could be more user friendly. Feel free to add an issue or PR on the webpage repo with some suggestions.